JASP-Engine/JASP/R/correlationbayesian.R

```diff
@@ -254,42 +248,19 @@ CorrelationBayesian <- function(dataset=NULL, options, perform="run",
 	# State: Processing
 	#
 	if (!is.null(state)) {
-		# Retrieve from state, I could have done this directly on state$footnotes etc, 
-		# but then I have to check whether it is initialised and whether I can write to it
-		# for instance, in the writing to bfValuesList
-		#
+		# Retrieve from state, 
 		# Pairwise based on names
-		rValuesListExcludePairwise <- state$rValuesExcludePairwise
-		bfValuesListExcludePairwise <- state$bfValuesExcludePairwise
-		footnotesListExcludePairwise <- state$footnotesExcludePairwise
+		memoryExcludePairwise <- state$memoryExcludePairwise
 		
-		# Pairwise based on n and r
-		bfValuesListExcludeListwise <- state$bfValuesExcludeListwise
-		footnotesListExcludeListwise <- state$footnotesExcludeListwise
+		# Listwise based on n and r
+		memoryExcludeListwise <- state$memoryExcludeListwise
 	} else {
-		# State: Create the hierarchical structure of the state object
-	    # Initialise if there is no state
+		# State is null
+	    # Initialise: the hierarchical structure of the state object
 		temp <- list()
-		for (levels in 1:3) {
-			temp <- list(temp)
-			
-			if (levels==2) {
-				# Structure is [[variableName]][[columnName]]
-				rValuesListExcludePairwise <- temp
-				footnotesListExcludePairwise <- temp
-				
-				# Not necessary to consider the names only need the ns
-				# Structure is [[nLabel]][[rLabel]]
-				footnotesListExcludeListwise <- temp
-			} else if (levels==3) {
-				# Structure is [[priorLabel]][[variableName]][[columnName]]
-				bfValuesListExcludePairwise <- temp 
-				
-				# Not necessary to consider the names only need the rs
-				# Structure is [[priorLabel]][[nLabel]][[rLabel]]
-				bfValuesListExcludeListwise <- temp
-			}
-		}
+		
+		memoryExcludePairwise <- list(list(list(list())))
```

Comment:
add:
\# future task: add list to distinguish between different correlation coefficients, then structure becomes [[corCoefficient]][[priorLabel]][[variableName]][[columnName]]